src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -700,100 +753,119 @@
   "fluid.tconstruct.molten_tin": "熔融锡",
   "fluid.tconstruct.flowing_molten_tin": "流动的熔融锡",
   "item.tconstruct.molten_tin_bucket": "熔融锡桶",
+  "fluid.tconstruct.molten_tin.fluid_effect": "他们说多吃胡萝卜对眼睛好，但熔融锡对眼睛更好。",
 
   "block.tconstruct.molten_aluminum_fluid": "熔融铝",
   "fluid.tconstruct.molten_aluminum": "熔融铝",
   "fluid.tconstruct.flowing_molten_aluminum": "流动的熔融铝",
   "item.tconstruct.molten_aluminum_bucket": "熔融铝桶",
+  "fluid.tconstruct.molten_aluminum.fluid_effect": "熔融铝像奶一样，但更争强好胜。可以考虑用它来应对有许多效果的对手。",
 
   "block.tconstruct.molten_lead_fluid": "熔融铅",
   "fluid.tconstruct.molten_lead": "熔融铅",
   "fluid.tconstruct.flowing_molten_lead": "流动的熔融铅",
   "item.tconstruct.molten_lead_bucket": "熔融铅桶",
+  "fluid.tconstruct.molten_lead.fluid_effect": "如果你愿意等，熔融铅会给你的攻击带来一点额外的提升。",
 
   "block.tconstruct.molten_silver_fluid": "熔融银",
   "fluid.tconstruct.molten_silver": "熔融银",
   "fluid.tconstruct.flowing_molten_silver": "流动的熔融银",
   "item.tconstruct.molten_silver_bucket": "熔融银桶",
+  "fluid.tconstruct.molten_silver.fluid_effect": "熔融银可以排出毒素而不波及正面效果。",
 
   "block.tconstruct.molten_nickel_fluid": "熔融镍",
   "fluid.tconstruct.molten_nickel": "熔融镍",
   "fluid.tconstruct.flowing_molten_nickel": "流动的熔融镍",
   "item.tconstruct.molten_nickel_bucket": "熔融镍桶",
+  "fluid.tconstruct.molten_nickel.fluid_effect": "熔融镍可以攻击敌人并降低反击风险，这种流体有什么理由不爱呢？",
 
   "block.tconstruct.molten_zinc_fluid": "熔融锌",
   "fluid.tconstruct.molten_zinc": "熔融锌",
   "fluid.tconstruct.flowing_molten_zinc": "流动的熔融锌",
   "item.tconstruct.molten_zinc_bucket": "熔融锌桶",
+  "fluid.tconstruct.molten_zinc.fluid_effect": "熔融锌可能是提升速度的最佳选择，你的速度让对手望尘莫及。",
 
   "block.tconstruct.molten_platinum_fluid": "熔融铂",
   "fluid.tconstruct.molten_platinum": "熔融铂",
   "fluid.tconstruct.flowing_molten_platinum": "流动的熔融铂",
   "item.tconstruct.molten_platinum_bucket": "熔融铂桶",
+  "fluid.tconstruct.molten_platinum.fluid_effect": "熔融铂对敌人的挖掘速度削减作用尚不明晰，但其他功能仍待发掘。",
 
   "block.tconstruct.molten_tungsten_fluid": "熔融钨",
   "fluid.tconstruct.molten_tungsten": "熔融钨",
   "fluid.tconstruct.flowing_molten_tungsten": "流动的熔融钨",
   "item.tconstruct.molten_tungsten_bucket": "熔融钨桶",
+  "fluid.tconstruct.molten_tungsten.fluid_effect": "熔融钨是重金属之一，希望它能让敌人定在那里。",
 
   "block.tconstruct.molten_osmium_fluid": "熔融锇",
   "fluid.tconstruct.molten_osmium": "熔融锇",
   "fluid.tconstruct.flowing_molten_osmium": "流动的熔融锇",
   "item.tconstruct.molten_osmium_bucket": "熔融锇桶",
+  "fluid.tconstruct.molten_osmium.fluid_effect": "熔融锇可以定住敌人，任你处置。",
 
   "block.tconstruct.molten_uranium_fluid": "熔融铀",
   "fluid.tconstruct.molten_uranium": "熔融铀",
   "fluid.tconstruct.flowing_molten_uranium": "流动的熔融铀",
   "item.tconstruct.molten_uranium_bucket": "熔融铀桶",
+  "fluid.tconstruct.molten_uranium.fluid_effect": "熔融铀产生的放射性衰变可不安全，所以把它喷在你的敌人身上。",
```

```suggestion
  "fluid.tconstruct.molten_uranium.fluid_effect": "熔融铀产生的放射性衰变可不安全，所以把它喷到你的敌人身上吧。",
```
Polishing, not required.